R/nanotime.R

```diff
@@ -339,6 +339,9 @@ setAs("nanotime", "POSIXlt", function(from) as.POSIXlt.nanotime(from))
 ##' @rdname nanotime
 as.Date.nanotime <- function(x, ...) {
     args <- list(...)
+    if (length(args) == 0) {
+        args <- list(tz="UTC")
```

Comment:
I think line 343 could replace the `stop`. That is, if we don't find `tz` then we add it, so no exception. We could also potentially be stricter as the ellipsis for this case should either be no argument or `tz`, but other argument names are not supported, so we might want to raise an exception if the user calls `as.Date(some_nanotime, foo=1)`.

Reply:
I committed the above so do you want to just commit another refining iteration?  When I wrote it was mostly as a 'how about ...' to raise an argument.